msgid ""
msgstr ""
"Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-07-13 20:53+0200\n"
"PO-Revision-Date: 2016-02-09 23:35+0100\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: German\n"
"Language: de_DE\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Poedit 1.8.7\n"
"X-Crowdin-Project: pyload\n"
"X-Crowdin-File: /pyLoad/core.pot\n"

#: pyLoadCore.py:181
msgid "Received Quit signal"
msgstr "Signal zum Beenden erhalten"

#: pyLoadCore.py:302
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad ist bereits gestartet mit der pid %s"

#: pyLoadCore.py:316
#, python-format
msgid "Failed changing group: %s"
msgstr "Fehler beim Ändern der Gruppe: %s"

#: pyLoadCore.py:326
#, python-format
msgid "Failed changing user: %s"
msgstr "Fehler beim Ändern des Benutzers: %s"

#: pyLoadCore.py:328
msgid "folder for logs"
msgstr "Ordner für Logs"

#: pyLoadCore.py:339
msgid "Starting"
msgstr "starte"

#: pyLoadCore.py:340
#, python-format
msgid "Using home directory: %s"
msgstr "Benutze Home-Verzeichnis: %s"

#: pyLoadCore.py:349
msgid "pycrypto to decode container files"
msgstr "pyCrypto um Containerdateien zu öffnen"

#: pyLoadCore.py:352
msgid "folder for temporary files"
msgstr "Ordner für temporäre Dateien"

#: pyLoadCore.py:357
msgid "folder for downloads"
msgstr "Downloadordner"

#: pyLoadCore.py:360
msgid "OpenSSL for secure connection"
msgstr "OpenSSL für eine sichere Verbindung"

#: pyLoadCore.py:364
msgid "Moving old user config to DB"
msgstr "Migriere alte Benutzereinstellungen in die Datenbank..."

#: pyLoadCore.py:367
msgid "Please check your logindata with ./pyLoadCore.py -u"
msgstr "Bitte überprüfe deine Login-Informationen mit './pyLoadCore.py -u'"

#: pyLoadCore.py:370
msgid "All links removed"
msgstr "Alle Links entfernt"

#: pyLoadCore.py:401
#, python-format
msgid "Downloadtime: %s"
msgstr "Downloadzeit: %s"

#: pyLoadCore.py:411
#, python-format
msgid "Free space: %s"
msgstr "Freier Speicher: %sGB"

#: pyLoadCore.py:431
msgid "Activating Accounts..."
msgstr "Aktiviere Konten..."

#: pyLoadCore.py:437
msgid "Activating Plugins..."
msgstr "Aktiviere Plugins..."

#: pyLoadCore.py:440
msgid "pyLoad is up and running"
msgstr "pyLoad ist gestartet und läuft"

#: pyLoadCore.py:459
msgid "restarting pyLoad"
msgstr "starte pyLoad neu"

#: pyLoadCore.py:463
msgid "pyLoad quits"
msgstr "pyLoad wird beendet"

#: pyLoadCore.py:520
#, python-format
msgid "Install %s"
msgstr "Installiere %s"

#: pyLoadCore.py:556
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "konnte %(desc)s nicht finden: %(name)s"

#: pyLoadCore.py:558
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "konnte %(desc)s nicht erstellen: %(name)s"

#: pyLoadCore.py:579
msgid "shutting down..."
msgstr "Beende..."

#: pyLoadCore.py:596
msgid "error while shutting down"
msgstr "Fehler beim Beenden"

#: pyLoadCore.py:660
msgid "killed pyLoad from Terminal"
msgstr "pyLoad vom Terminal beendet"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "Link Datenbank wurde aufgrund inkompatibler Version gelöscht."

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "Link Datenbank konnte nicht konvertiert werden."

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "Datenbank wurde von v2 nach v3 konvertiert."

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "Datenbank wurde von v3 nach v4 konvertiert."

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Konvertiere alte Django DB"

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "Fertig"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr "Offline"

#: module/database/FileDatabase.py:45
msgid "online"
msgstr "Online"

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "Eingereiht"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "Übersprungen"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "Warte"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "Vorübergehend offline"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "Starte"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "Fehlgeschlagen"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "Abgebrochen"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "Entschlüssle"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "Benutzerdefiniert"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "Lade herunter"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "Verarbeite"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "Unbekannt"

#: module/database/FileDatabase.py:531 module/plugins/hooks/IRCInterface.py:74
#: module/plugins/hooks/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Paket fertiggestellt: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Nutze SSL Thrift Backend"

#: module/remote/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Fehler beim Fernzugriff: %s"

#: module/remote/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Starte %(name)s: %(addr)s:%(port)s"

#: module/remote/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Konnte Backend %(name)s nicht laden | %(error)s"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "Warte %s"

#: module/web/ServerThread.py:36
msgid "SSL certificates not found."
msgstr "SSL Zertifikat nicht gefunden."

#: module/web/ServerThread.py:40
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Entschuldigung! Wir haben den Support für das Starten von %s direkt in pyLoad eingestellt"

#: module/web/ServerThread.py:41
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Du kannst den multithreaded-server benutzen, welcher eine gute Performance, SSL und"

#: module/web/ServerThread.py:42
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "natürlich kannst du weiterhin deine existierende %s mit pyLoads fastcgi server benutzen"

#: module/web/ServerThread.py:43
msgid "sample configs are located in the module/web/servers directory"
msgstr "Beispiel-Konfigurationsdateien befinden sich in dem Verzeichnis 'module/web/servers'"

#: module/web/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "Kann %(server)s nicht benutzen, weil python-flup nicht installiert ist!"

#: module/web/ServerThread.py:58
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Fehler beim Laden vom lightweight Server: %s"

#: module/web/ServerThread.py:59
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Du musst 'bjoern' herunterladen und kompilieren, https://github.com/jonashaag/bjoern"

#: module/web/ServerThread.py:60
msgid "Copy the boern.so to module/lib folder or use setup.py install"
msgstr "Kopiere 'boern.so' in 'module/lib' oder benutze 'setup.py install'"

#: module/web/ServerThread.py:61
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Natürlich musst musst etwas Ahnung von Linux haben und wie man Software kompiliert"

#: module/web/ServerThread.py:64
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Der 'threaded' Server wird wegen Performance Problemen unter Windows benutzt."

#: module/web/ServerThread.py:80 module/web/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "SSL funktioniert mit diesem Server nicht, benutze bitte alternativ den 'threaded' Server"

#: module/web/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Starte den eingebauten Webserver: %(host)s:%(port)d"

#: module/web/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Starte den threaded SSL Webserver: %(host)s:%(port)d"

#: module/web/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Starte den threaded Webserver: %(host)s:%(port)d"

#: module/web/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Starte fastcgi Webserver: %(host)s:%(port)d"

#: module/web/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Starte bjoern Webserver: %(host)s:%(port)d"

#: module/web/pyload_app.py:127
msgid "You dont have permission to access this page."
msgstr "Sie haben nicht die Berechtigung diese Seite zu besuchen."

#: module/web/pyload_app.py:195
msgid "Download directory not found."
msgstr "Downloadverzeichnis nicht gefunden."

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "unlimited"
msgstr "unbegrenzt"

#: module/web/pyload_app.py:264 module/web/pyload_app.py:271
msgid "not available"
msgstr "nicht verfügbar"

#: module/web/pyload_app.py:509
msgid "Run pyLoadCore.py -s to access the setup."
msgstr "Starte pyLoadCore.py -s um auf dieses Setup zuzugreifen."

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Eine Download Verbindung ist fehlggeschlagen, Falle zurück auf eine Verbindung | %s"

#: module/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Starte Download: %s"

#: module/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Download fertiggestellt: %s"

#: module/PluginThread.py:194 module/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "Dem Plugin %s fehlt eine Funktion."

#: module/PluginThread.py:202 module/PluginThread.py:265
#: module/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Download abgebrochen: %s"

#: module/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Download erneut gestartet: %(name)s | %(msg)s"

#: module/PluginThread.py:231 module/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "Download ist nicht verfügbar: %s"

#: module/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "Download ist vorübergehend nicht verfügbar: %s"

#: module/PluginThread.py:237 module/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Download fehlgeschlagen: %(name)s | %(msg)s"

#: module/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "Verbindungsaufbau zum Host fehlgeschlagen oder die Verbindung wurde zurückgesetzt. Erneuter Versuch in 1 Minute..."

#: module/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Download %(name)s wegen %(plugin)s übersprungen"

#: module/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Decrypting startet: %s"

#: module/PluginThread.py:376 module/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Decrypting fehlgeschlagen: %(name)s | %(msg)s"

#: module/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Wiederhole %s"

#: module/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Holen von Informationen für %(name)s fehlgeschlagen | %(err)s"

#: module/HookManager.py:90 module/plugins/Hook.py:103
#, python-format
msgid "Error executing hooks: %s"
msgstr "Fehler beim ausführen des Hook: %s"

#: module/HookManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Konnte %(name)s nicht aktivieren"

#: module/HookManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Aktivierte Plugins: %s"

#: module/HookManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Deaktivierte Plugins: %s"

#: module/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "erneute Verbindung fehlgeschlagen: %s"

#: module/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Skript für erneute Verbindung nicht gefunden!"

#: module/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "versuche erneute Verbindung"

#: module/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Fehler beim Ausführen des Skriptes für die erneute Verbindung!"

#: module/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Verbindung erneut aufgebaut, IP: %s"

#: module/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Auf dem Gerät ist nicht mehr genügend Speicherplatz vorhanden"

#: module/plugins/Account.py:85 module/plugins/Account.py:90
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Fehler beim Anmelden mit Konto %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Falsches Passwort"

#: module/plugins/Account.py:243
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "Ihre Zeit %s hat ein falsches Format, nutzen Sie dieses: 1:22-3:44"

#: module/plugins/Account.py:269
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "Konto %s hat nicht genügend Traffic, erneuter Versuch in 30 min"

#: module/plugins/Account.py:276
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "Konto %s ist abgelaufen, erneuter Versuch in 1Std"

#: module/plugins/crypter/SerienjunkiesOrg.py:126
msgid "Downloadlimit reached"
msgstr "Downloadlimit erreicht"

#: module/plugins/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s hat ein ungültiges Muster."

#: module/plugins/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Fehler beim Importieren von %(name)s: %(msg)s"

#: module/plugins/internal/MultiHoster.py:132
msgid "No Hoster loaded"
msgstr "Kein Hoster geladen"

#: module/plugins/accounts/BitshareCom.py:38
msgid "Activate direct Download in your Bitshare Account"
msgstr "Aktiviere direkten Download in deinem Bitshare Account"

#: module/plugins/container/LinkList.py:64
msgid "LinkList could not be cleared."
msgstr "links.txt konnte nicht geleert werden."

#: module/plugins/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Account-Einstellungen aufgrund neuem Konfigurationsformat gelöscht."

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Authorisierung erforderlich (Benutzername:Passwort)"

#: module/plugins/hoster/SimplydebridCom.py:23
#: module/plugins/hoster/RealdebridCom.py:40
#: module/plugins/hoster/FreeWayMe.py:39 module/plugins/hoster/ZeveraCom.py:21
#: module/plugins/hoster/UnrestrictLi.py:52
#: module/plugins/hoster/Vipleech4uCom.py:30
#: module/plugins/hoster/Premium4Me.py:27 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/SimplyPremiumCom.py:52
#: module/plugins/hoster/MegaDebridEu.py:46
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/LinksnappyCom.py:29
#: module/plugins/hoster/DebridItaliaCom.py:39
#: module/plugins/hoster/RPNetBiz.py:28
#: module/plugins/hoster/MultiDebridCom.py:40
#: module/plugins/hoster/ReloadCc.py:26 module/plugins/hoster/OverLoadMe.py:38
#: module/plugins/hoster/RehostTo.py:25
#: module/plugins/hoster/PremiumizeMe.py:24 module/plugins/hooks/RPNetBiz.py:45
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Bitte gib deine %s Kontodaten ein oder deaktiviere dieses Plugin"

#: module/plugins/hoster/FilesMailRu.py:98
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Es gab HTML-Code in der heruntergeladen Datei (%s)... Umleitungsfehler? Der Download wird neu gestartet."

#: module/plugins/hoster/NetloadIn.py:145
#: module/plugins/hoster/NetloadIn.py:169
msgid "File temporarily not available"
msgstr "Datei zur Zeit nicht verfügbar"

#: module/plugins/hoster/NetloadIn.py:182
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: warte zwischen den Downloads %d s."

#: module/plugins/hoster/NetloadIn.py:213
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: Warte auf Captcha %d s."

#: module/plugins/hoster/NetloadIn.py:251
msgid "Downloaded File was empty"
msgstr "Die heruntergeladene Datei ist leer"

#: module/plugins/hoster/MegaDebridEu.py:49
#, python-format
msgid "Impossible to connect to %s"
msgstr "%s kann nicht verbunden werden"

#: module/plugins/hoster/MegaDebridEu.py:89
#, python-format
msgid "Impossible to debrid %s"
msgstr "Analyse von %s nicht möglich"

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "API Schlüssel ungültig"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Nicht genügend Traffic übrig"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Traffic überschritten"

#: module/plugins/hoster/RapidshareCom.py:99
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Traffic Share (direkter Download)"

#: module/plugins/hoster/RapidshareCom.py:126
#: module/plugins/hoster/RapidshareCom.py:193
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Von dieser IP Adresse wird bereits heruntergeladen, warte 60 Sekunden"

#: module/plugins/hoster/RapidshareCom.py:130
msgid "Invalid Auth Code, download will be restarted"
msgstr "Ungültiger Authorization-Code, Download wird neugestartet"

#: module/plugins/hoster/RapidshareCom.py:198
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: Keine freien Slots"

#: module/plugins/hoster/RapidshareCom.py:201
msgid "You need a premium account for this file"
msgstr "Du benötigst ein Premium-Konto für diese Datei"

#: module/plugins/hoster/RapidshareCom.py:203
msgid "Filename reported invalid"
msgstr "Dateiname als ungültig gemeldet"

#: module/plugins/hoster/FileserveCom.py:100
msgid "Parallel download error, now waiting 60s."
msgstr "Parallel-Download Fehler, warte 60s."

#: module/plugins/hoster/FileserveCom.py:216
msgid "Not logged in."
msgstr "Nicht eingeloggt."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Entschlüsselung fehlgeschlagen"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Für die Datei wurde kein Schlüssel in der URL übertragen"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Fehler Code:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr "Datei existiert nicht."

#: module/plugins/hooks/UpdateManager.py:97
msgid "Not able to connect server to get updates"
msgstr "Server kann für Updates nicht erreicht werden"

#: module/plugins/hooks/UpdateManager.py:111
msgid "No pyLoad version available"
msgstr "Es ist keine neue pyload Version verfügbar"

#: module/plugins/hooks/UpdateManager.py:118
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Neue pyLoad Version %s verfügbar  ***"

#: module/plugins/hooks/UpdateManager.py:119
msgid "***  Get it here: https://github.com/pyload/pyload/releases  ***"
msgstr "*** Hier beziehen: https://github.com/pyload/pyload/releases ***"

#: module/plugins/hooks/UpdateManager.py:170
#, python-format
msgid "New version of [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"
msgstr "Neue Version von [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"

#: module/plugins/hooks/UpdateManager.py:180
#: module/plugins/hooks/UpdateManager.py:185
#, python-format
msgid "Error when updating plugin %s"
msgstr "Fehler beim Update des Plugins %s"

#: module/plugins/hooks/UpdateManager.py:185
msgid "Version mismatch"
msgstr "Versionskonflikt"

#: module/plugins/hooks/UpdateManager.py:196
#, python-format
msgid "Removed blacklisted plugin: [%(type)s] %(name)s"
msgstr "Gesperrtes Plugin entfernt: [%(type)s] %(name)s"

#: module/plugins/hooks/UpdateManager.py:204
msgid "Plugins updated and reloaded"
msgstr "Plugins aktualisiert und neu geladen"

#: module/plugins/hooks/UpdateManager.py:206
msgid "*** Plugins have been updated, pyLoad will be restarted now ***"
msgstr "*** Plugins wurden aktualisiert, pyload startet nun neu ***"

#: module/plugins/hooks/UpdateManager.py:211
msgid "No plugin updates available"
msgstr "Keine Plugin-Updates verfügbar"

#: module/plugins/hooks/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Download fertig: %(name)s @ %(plugin)s "

#: module/plugins/hooks/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Neue Captcha Anfrage: %s"

#: module/plugins/hooks/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Beantworte Captcha mit \"c %s Text\""

#: module/plugins/hooks/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Bitte füge erst dein premium.to Konto hinzu und starte pyLoad neu"

#: module/plugins/hooks/HotFolder.py:81
#, python-format
msgid "Added %s from HotFolder"
msgstr "%s aus HotFolder hinzugefügt"

#: module/plugins/hooks/ExtractArchive.py:94
#, python-format
msgid "No %s installed"
msgstr "Kein %s installiert"

#: module/plugins/hooks/ExtractArchive.py:96
#: module/plugins/hooks/ExtractArchive.py:101
#, python-format
msgid "Could not activate %s"
msgstr "Konnte %s nicht aktivieren"

#: module/plugins/hooks/ExtractArchive.py:106
msgid "Activated"
msgstr "Aktiviert"

#: module/plugins/hooks/ExtractArchive.py:108
msgid "No Extract plugins activated"
msgstr "Keine Entpacken-Plugins aktiviert"

#: module/plugins/hooks/ExtractArchive.py:120
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Paket %s eingereiht für späteres entpacken"

#: module/plugins/hooks/ExtractArchive.py:143
#, python-format
msgid "Check package %s"
msgstr "Überprüfe Paket %s"

#: module/plugins/hooks/ExtractArchive.py:184
#, python-format
msgid "Extract to %s"
msgstr "Entpacke nach %s"

#: module/plugins/hooks/ExtractArchive.py:199
msgid "No files found to extract"
msgstr "Keine Dateien zum entpacken gefunden"

#: module/plugins/hooks/ExtractArchive.py:206
msgid "extracting"
msgstr "entpacke"

#: module/plugins/hooks/ExtractArchive.py:217
msgid "Password protected"
msgstr "Passwortgeschützt"

#: module/plugins/hooks/ExtractArchive.py:238
msgid "Wrong password"
msgstr "Falsches Passwort"

#: module/plugins/hooks/ExtractArchive.py:246
#, python-format
msgid "Deleting %s files"
msgstr "Lösche %s Dateien"

#: module/plugins/hooks/ExtractArchive.py:253
msgid "Extracting finished"
msgstr "Entpacken abgeschlossen"

#: module/plugins/hooks/ExtractArchive.py:259
msgid "Archive Error"
msgstr "Archiv-Fehler"

#: module/plugins/hooks/ExtractArchive.py:261
msgid "CRC Mismatch"
msgstr "CRC Fehler"

#: module/plugins/hooks/ExtractArchive.py:265
msgid "Unknown Error"
msgstr "Unbekannter Fehler"

#: module/plugins/hooks/ExtractArchive.py:317
msgid "Setting User and Group failed"
msgstr "Setzen von Benutzer und Gruppe fehlgeschlagen"

#: module/plugins/hooks/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: Port 9666 wird bereits verwendet"

#: module/plugins/hooks/CaptchaTrader.py:70
#: module/plugins/hooks/Captcha9kw.py:60
#: module/plugins/hooks/ExpertDecoders.py:51
#, python-format
msgid "%s credits left"
msgstr "%s Credits verbleibend"

#: module/plugins/hooks/CaptchaTrader.py:118
msgid "Could not send response."
msgstr "Konnte keine Antwort senden."

#: module/plugins/hooks/CaptchaTrader.py:136
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "Dein CaptchaTrader Konto hat nicht genügend Credits"

#: module/plugins/hooks/LinkdecrypterCom.py:45
msgid "Crypter list not found"
msgstr "Liste von Cryptern nicht gefunden"

#: module/plugins/hooks/LinkdecrypterCom.py:59
msgid "Crypter list is empty"
msgstr "Liste von Cryptern ist leer"

#: module/plugins/hooks/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Download abgeschlossen: %(name)s @ %(plugin)s"

#: module/plugins/hooks/Captcha9kw.py:94
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Neue CaptchaID von Upload: %s : %s"

#: module/plugins/hooks/Captcha9kw.py:130
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "Dein Captcha 9kw.eu Account hat nicht genügend Credits"

#: module/plugins/hooks/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Installierte Skripte für %s: "

#: module/plugins/hooks/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script nicht ausführbar:"

#: module/plugins/hooks/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Fehler in %(script)s: %(error)s"

#: module/plugins/hooks/ExpertDecoders.py:96
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Dein ExpertDecoders Account hat nicht genügend Credits"

#: module/plugins/hooks/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Bitte fügen Sie zuerst Ihr rehost.to Konto hinzu und starten Sie pyLoad danach neu"

#: module/plugins/hooks/PremiumizeMe.py:50
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Bitte füge erst ein gültiges premiumize.me Konto hinzu und starte pyLoad neu."

#: module/plugins/hooks/CaptchaBrotherhood.py:70
#, python-format
msgid "%d credits left"
msgstr "%d Credits verbleibend"

#: module/plugins/Plugin.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil und Tesseract sind nicht installiert und kein Client ist verbunden für Captcha-Entzifferung"

#: module/plugins/Plugin.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Es konnte in angemessener zeit kein Captcha-Ergebnis durch dies Plugins ermittelt werden."

#: module/plugins/Plugin.py:498 module/plugins/Plugin.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Benutzer und Gruppe setzen fehlgeschlagen: %s"

#: module/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "Kein Client verbunden zum eingeben des Captchas"

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Paket %(name)s mit %(count)d Links hinzugefügt"

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "%(count)d Links wurden zum Paket #%(package)d hinzugefügt "

#: module/common/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "Keine JS Engine erkannt. Bitte installiere entweder Spidermonkey, ossp-js, pyv8 oder rhino"
